compiler/next/include/chpl/types/Param.h

```diff
@@ -97,7 +97,9 @@ class Param {
 
   // helper function to convert a value to a string
   static std::string valueToString(UniqueString v) {
-    return v.toString();
+    std::ostringstream ss;
```

Comment:
weird indenting in this function. But it should just be calling `v.str()` since that is the way to convert a UniqueString to a `std::string`.